console/src/components/UIKit/atoms/SwitchButton/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 
 import { StyledSwitchButton, StyledSlider } from './SwitchButton';
 
-export const SwitchButton = props => {
+export const SwitchButton: React.FC = props => {
```

Comment:
`SwicthButton` is accepting props: https://github.com/hasura/graphql-engine/blob/76e1e0286c441ec84001f3871750c22a40c1d504/console/src/components/UIKit/atoms/SwitchButton/index.tsx#L10 so we need to do the same thing as in the case of `Spinner` component.

Reply:
I fix it.

https://github.com/hasura/graphql-engine/pull/4504/commits/67e6487920e8eff6e9f096538741f05713f71824